client/app/lib/remote-extensions/index.coffee

```diff
@@ -21,11 +21,13 @@ module.exports = RemoteExtensions =
 
     (Object.keys remote.api).forEach (model) ->
 
+      original = remote.api[model]::init
+
       # overriding ::init on all `remote.api` which is getting called
       # inside of it's constructor.
       remote.api[model]::init = (data) ->
 
-        super
+        original.call(this, data)
```

Comment:
It was throwing an error saying `you can't use super out of class context`, this was the easiest fix for moving on.